client/network/src/config.rs

```diff
@@ -91,6 +92,9 @@ where
 	/// Instance of chain sync implementation.
 	pub chain_sync: Box<dyn ChainSync<B>>,
 
+	/// Interface that can be used to delegate syncing-related function calls to `ChainSync`
+	pub chain_sync_service: Arc<dyn ChainSyncInterface<B>>,
```

Comment:
Does it need to be an `Arc`? I don't see it shared anywhere so maybe `Box` would work?